### ISSUE TITLE
This tool shows that it supports solidity, does it not actually support it?

### ISSUE THREAD

**Issue by Care-C** [joernio/joern#4264]
![65e7fec41f74f62950b87c754603781f](https://github.com/joernio/joern/assets/61672270/19914c7a-52b8-4315-a0d6-68bc414fb77c)
![898d9493d7f413e91d759991925edf78](https://github.com/joernio/joern/assets/61672270/d361a0d3-930e-44f4-8845-9649a26ff344)
![9073ada0bf222750e5fed53fba185563](https://github.com/joernio/joern/assets/61672270/3d1acab7-7390-47fa-aa30-99a32ae06dd1)

**Comment by ycclnn** [comment]
No, only the frontends you can find under joerncli/frontends are supported.
